### ISSUE TITLE
Import fails with ValueError: source code string cannot contain null bytes

### ISSUE THREAD

**Issue by glaubitz** [oracle/oci-python-sdk#241]
I just tried upating the ```oci-python-sdk``` package in openSUSE to version 2.12.2 and while testing the package, I ran into this problem when issuing ```import oci``` in the Python3 interpreter:

```
glaubitz@suse-laptop:~> python3
Python 3.8.2 (default, Mar 05 2020, 18:58:42) [GCC] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import oci
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.8/site-packages/oci/__init__.py", line 4, in <module>
    from . import analytics, announcements_service, apigateway, application_migration, audit, autoscaling, bds, budget, container_engine, core, data_catalog, data_flow, data_science, database, dns, dts, email, events, file_storage, functions, healthchecks, identity, integration, key_management, limits, load_balancer, marketplace, monitoring, nosql, object_storage, oce, oda, ons, os_management, resource_manager, resource_search, secrets, streaming, vault, waas, work_requests
  File "/usr/lib/python3.8/site-packages/oci/bds/__init__.py", line 7, in <module>
    from .bds_client import BdsClient
  File "/usr/lib/python3.8/site-packages/oci/bds/bds_client.py", line 14, in <module>
    from .models import bds_type_mapping
  File "/usr/lib/python3.8/site-packages/oci/bds/models/__init__.py", line 24, in <module>
    from .work_request_error import WorkRequestError
ValueError: source code string cannot contain null bytes
>>>
```

**Comment by bhagwatvyas** [comment]
Hi @glaubitz, I tried importing oci 2.12.2 on openSUSE docker container (leap and tumbleweed) but am not able to reproduce the issue. Are you facing this issue only on Python 3 or Python 2 as well?

One of the reasons that could cause this issue is corrupted `.pyc` files for the oci package. Could you try deleting them to see if it fixes the issue?

**Comment by glaubitz** [comment]
Hi @bhagwatvyas!

I'm the maintainer of the OCI packages in openSUSE/SLE and therefore I continuously monitor the functionality of the packages while I'm updating them.

The problem first showed when I updated the package from version 2.11.0 to 2.12.2 which is why I never submitted the package to the archives.

However, today I updated the package to 2.13.0 and the problem vanished. The issue affected both Python 2 and Python 3, but as of recently, openSUSE Tumbleweed no longer ships Python 2 packages so I didn't test for Python 2. We do still ship Python 2 in SLE-12/SLE-15 and openSUSE Leap.

FWIW, I didn't install the OCI SDK through pip but all my tests are with actual RPMs that I created. Testing with pip wouldn't be of much use as I need to make sure the RPM packages work which provide to our (Public Cloud) customers, also in OCI.
